wallet/service.go

```diff
@@ -195,6 +199,19 @@ func SetupService(ctx context.Context, r *chi.Mux) (*chi.Mux, context.Context, *
 	if err != nil {
 		logger.Fatal().Err(err).Msg("failed to initialize wallet service")
 	}
+
+	// setup reputation client
+	var reputationClient reputation.Client
+	if os.Getenv("ENV") != "local" || len(os.Getenv("REPUTATION_SERVER")) > 0 {
```

Comment:
i believe these checks are already in the new fn

Reply:
updated in https://github.com/brave-intl/bat-go/pull/541/commits/4a6999e3d9af426bc5ac753a8fa816662c03bea0